  2.9% (1/35) 2016_12_02_0101_s00_01_g8b.png reg/2016_12_02_0101_s00_01_g8b.tif plain copy
  5.7% (2/35) reg/2016_12_02_0101_s00_01_g8b.tif 2016_12_02_0101_s00_02_g8b.png 199 96.828557 ['96.828557'] 96.828557
  8.6% (3/35) reg/2016_12_02_0101_s00_02_g8b.tif 2016_12_02_0101_s00_03_g8b.png 199 83.403684 ['83.403684'] reg/2016_12_02_0101_s00_02_g8b.log//TransformParameters.0.txt 199 83.403684 ['83.403684'] 83.403684
 11.4% (4/35) reg/2016_12_02_0101_s00_03_g8b.tif 2016_12_02_0101_s00_04_g8b.png 199 41.603052 ['41.603052'] reg/2016_12_02_0101_s00_03_g8b.log//TransformParameters.0.txt
Traceback (most recent call last):
  File "/elastix_scripts/recRegStack.py", line 309, in <module>
    main()
  File "/elastix_scripts/recRegStack.py", line 101, in main
    register(FNs[start:], FNo, args, FNp) # skip upto start
  File "/elastix_scripts/recRegStack.py", line 258, in register
    selx[i].Execute()
  File "/opt/SimpleElastix/lib/python/SimpleITK/SimpleITK.py", line 10996, in Execute
    return _SimpleITK.ElastixImageFilter_Execute(self)
RuntimeError: Exception thrown in SimpleITK ElastixImageFilter_Execute: /SimpleElastix/Code/Elastix/src/sitkElastixImageFilterImpl.cxx:259:
sitk::ERROR: 
itk::ExceptionObject (0x560749b41cb0)
Location: "unknown" 
File: /selx_build/Elastix/Core/Main/elxElastixFilter.hxx
Line: 253
Description: itk::ERROR: Self(0x560749d74da0): Internal elastix error: See elastix log (use LogToConsoleOn() or LogToFileOn()).


